CHANGELOG.md

```diff
@@ -1,3 +1,46 @@
+## 1.3.0 (November 8, 2023)
+
+SECURITY:
+
+* Update Envoy version to 1.25.11 to address [CVE-2023-44487](https://github.com/envoyproxy/envoy/security/advisories/GHSA-jhv4-f7mr-xx76) [[GH-3116](https://github.com/hashicorp/consul-k8s/issues/3116)]
+
+FEATURES:
+
+* :tada: This release provides the ability to preview Consul's v2 Catalog and Resource API if enabled.
+The new model supports multi-port application deployments with only a single Envoy proxy.
+Note that the v1 and v2 catalogs are not cross compatible, and not all Consul features are available within this v2 feature preview.
+See the [v2 Catalog and Resource API documentation](https://developer.hashicorp.com/consul/docs/k8s/multiport) for more information.
+The v2 Catalog and Resources API should be considered a feature preview within this release and should not be used in production environments.
+
+### Limitations
+* The v1 and v2 catalog APIs cannot run concurrently.
+* The Consul UI must be disable. It does not support multi-port services or the v2 catalog API in this release.
+* HCP Consul does not support multi-port services or the v2 catalog API in this release.
+
```

Comment:
Nit, whould this have a heading for ###Significant Pull Requests 

like we did in the Core repo? https://github.com/hashicorp/consul/releases/tag/v1.17.0 

If so may need to be backported or changed elsewhere.

Reply:
Will handle this in a separate PR since it will need to be backported and such